OpenDreamShared/Compiler/DM/DMParser.cs

```diff
@@ -921,6 +921,9 @@ public class DMParser : Parser<Token> {
 
                 if (Check(TokenType.DM_Comma)) {
                     Whitespace();
+                    
+                    if (Check(TokenType.DM_IndeterminateArgs)) break;
```

Comment:
This doesn't work with `proc/example(a, ..., b)`

Reply:
That's valid DM?

Reply:
Yes

Reply:
While this is technically valid, the docs specify the last parameter can end in `...` and AFAIK nobody uses it anywhere except as the last arg. Do you still want me to change it?

Reply:
I'd prefer for the compiler to accept it, for now. Maybe later we can make it a warning too.